packages/backend/migrations/20220819120358_create_payment_pointer_keys_table.js

```diff
@@ -1,8 +1,8 @@
 exports.up = function (knex) {
-  return knex.schema.createTable('clientKeys', function (table) {
+  return knex.schema.createTable('paymentPointerKeys', function (table) {
```

Comment:
@wilsonianb -my suggestion to @omertoast was to rename this so that we can differentiate between these keys and the keys we cache from incoming clients. i.e. Client keys would be keys we lookup/cache for clients making incoming requests and payment pointer keys are keys we have on record for our own payment pointers